From c6b52612d1ae0af5c72ff1a8a0e9dbe168a79fcb Mon Sep 17 00:00:00 2001
From: Yusuke Goda <yusuke.goda.sx@renesas.com>
Date: Thu, 17 Jan 2019 06:02:14 +0000
Subject: [PATCH 768/909] arm64: dts: r8a77965-m3nulcb: Add VSPM I/F driver
 node

commit b8d5de71addccb19c14f077625fdd562c52dd389 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Add VSP Manager I/F driver node for M3NULCB board on R8A77965 SoC.

Signed-off-by: Yusuke Goda <yusuke.goda.sx@renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77965-m3nulcb.dts | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77965-m3nulcb.dts b/arch/arm64/boot/dts/renesas/r8a77965-m3nulcb.dts
index f1d6c82a43f9..5158094feaad 100644
--- a/arch/arm64/boot/dts/renesas/r8a77965-m3nulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77965-m3nulcb.dts
@@ -55,6 +55,10 @@
 	mmngrbuf {
 		compatible = "renesas,mmngrbuf";
 	};
+
+	vspm_if {
+		compatible = "renesas,vspm_if";
+	};
 };
 
 &du {
@@ -67,3 +71,11 @@
 	clock-names = "du.0", "du.1", "du.3",
 		      "dclkin.0", "dclkin.1", "dclkin.3";
 };
+
+&vspb {
+	status = "okay";
+};
+
+&vspi0 {
+	status = "okay";
+};
-- 
2.17.1

